### DISCUSSION TITLE
Is there a command to bump SemVer version (e.g. patch, minor, major, alpha/prerelease)

### DISCUSSION THREAD

**Discussion by asmith26** [astral-sh/rye#1141] [Q&A] [answered]
Hi,

Is there a command to bump package version with rye e.g. `rye version patch`?

Thanks for any help!

**Comment by rmarquis** [comment, marked as answer]
Yes.

Beside an explicit version string (`rye version <version>`), a bump rule such as `patch`, `minor`, `major`
can be passed with `--bump` or `-b`.

**Reply by asmith26** [reply]
Many thanks for your help @rmarquis
